package.json

```diff
@@ -52,9 +52,8 @@
   },
   "scripts": {
     "browserstack": "sh build/run-browserstack.sh",
-    "ci": "grunt test",
-    "ci-include-lint": "grunt eslint && grunt test",
-    "test": "grunt",
+    "build": "grunt build",
+    "test": "grunt test",
```

Comment:
https://github.com/qunitjs/qunit/blob/master/test/index.html#L7

We still need to build the dist files for the tests, right?